src/content/docs/mobile-monitoring/new-relic-mobile-cordova-phonegap/get-started-with-cordova-monitoring.mdx

```diff
@@ -14,7 +14,7 @@ import mobileHybridSample from 'images/mobile_screenshot-full_hybrid-summary.web
 
 import mobileQueryExample from 'images/mobile_screenshot-full_query-example.webp'
 
-Our New Relic Cordova agent monitors your Capacitor mobile app and provides deep insights into your app's performance, errors, and user experience. Written in JavaScript, the Cordova agent includes all New Relic mobile monitoring features we offer for native mobile apps. Once you install and configure the Cordova agent, you'll be able to:
+Our New Relic Cordova agent monitors your Cordova mobile app and provides deep insights into your app's performance, errors, and user experience. Written in JavaScript, the Cordova agent includes all New Relic mobile monitoring features we offer for native mobile apps. Once you install and configure the Cordova agent, you'll be able to:
```

Comment:
Is there a reason to mention the Javascript? If not I think we can cut it safely.